components/espace-agent-components/agent-wall/documents.tsx

```diff
@@ -0,0 +1,31 @@
+import routes from '#clients/routes';
+import { IUniteLegale } from '#models/index';
+import AgentWall from '.';
+
+const AgentWallDocuments: React.FC<{
+  title: string;
+  id: string;
+  uniteLegale: IUniteLegale;
+}> = ({ uniteLegale, id, title }) => {
+  return (
+    <AgentWall
+      id={id}
+      title={title}
+      modalFooter={
+        <>
+          <b>Comment accéder à ces documents autrement ?</b>
```

Comment:
> I find font-size changes does not bring good content hierarchy

It's not really about `font-size` change, but more about using the proper tag that conveys the best semantic meaning to the text. In this case, for instance, this is clearly a title of the following section, and thus, we should use a `h<x>` tag. This is good practice for SEO, navigation, and accessibility.

You can, of course, change the appearance of the `hx` in this case if you like. Personally, I like the ability to decouple a tag from appearance that `styled-components` allows with the `as` prop. 

Lastly, the `b` tag is deprecated as it does not convey semantic meaning. It's better to use `strong`.

Source: https://developer.mozilla.org/fr/docs/web/HTML/Element/strong#b_ou_strong

Reply:
Ok makes more sense. I also understand better your implem of #838 

I tend to focus more on UX, maybe too much. Especially as these pages are not indexed by robot. But they could be visited

Reply:
I am going to add simple `<hn/>` and remove `<strong />`